release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/getLatestIstioCtl | sh -
+#
+detect_os() {
+  OS=$(uname)
+  case $OS in
+    'Linux')
+      OSEXT='linux'
+      FILENAME='istioctl'
+      ;;
+    'WindowsNT')
+      OSEXT='win.exe'
+      FILENAME='istioctl.exe'
+      ;;
+    'Darwin') 
+      OSEXT='osx'
+      FILENAME='istioctl'
+      ;;
+    *) ;;
+  esac
+}
+
+get_version() {
+  if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  ISTIO_VERSION=$(curl -L -s https://api.github.com/repos/istio/istio/releases/latest | \
+                  grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/")
+  fi
+}
+
+download_istioctl() {
+  NAME="istioctl-${OSEXT}"
+  URL="https://storage.googleapis.com/istio-artifacts/${ISTIO_VERSION}/artifacts/istioctl/${NAME}"
+  echo "${URL}"
+  echo "Downloading ${NAME} ..."  
+  curl -L "${URL}" -o "${FILENAME}"
+  BINDIR="$(pwd)"
+  echo "Add the istioctl client to your PATH environment variable, on a MacOS or Linux system:"
```

Comment:
this doesn't seem to work without a chmod +x

Reply:
Linkerd got it working with 2 step: 1 download script 2. add to path.  maybe because it was in bin dir in the first place?